### ISSUE TITLE
Toplevel definition native is defined twice when using facades with Scala 3

### ISSUE THREAD

**Issue by sbrunk** [scalapy/scalapy#243]
I'm trying to define a simple facade like in the docs:

```scala
import me.shadaj.scalapy.py

@py.native trait PyString extends py.Object {
  def count(subsequence: String): Int = py.native
}
```

But I'm running into the following error:

```
[error] 4 |  def count(subsequence: String): Int = py.native
[error]   |                                        ^
[error]   |Toplevel definition native is defined in
[error]   |  /.../repository/public/me/shadaj/scalapy-core_3/0.5.1%2B1-a19bda77/scalapy-core_3-0.5.1%2B1-a19bda77.jar(me/shadaj/scalapy/py/PyMacros.class)
[error]   |and also in
[error]   |  /.../repository/public/me/shadaj/scalapy-macros_3/0.5.1%2B1-a19bda77/scalapy-macros_3-0.5.1%2B1-a19bda77.jar(me/shadaj/scalapy/py/native.class)
[error]   |One of these files should be removed from the classpath.
```

The error only occurs when using Scala 3, it seems to work fine on 2.13.

**Comment by shadaj** [comment]
Ooh, this is interesting. It seems that the compiler can't disambiguate between the annotation class and the macro method. I'm surprised this doesn't show up when running our tests though, will investigate.

**Comment by sbrunk** [comment]
Thanks for looking into it @shadaj and let me know if I can help in any way.

Just in case, because it doesn't show up in the tests, here's a minimal project that should allow you to reproduce the error: https://github.com/sbrunk/scalapy-native-issue-repro

**Comment by sbrunk** [comment]
Hey @shadaj is there a chance you could look into this? It would be great to be able to use ScalaPy facades in Scala 3.

Still weird that the issue does not show up in the tests. I also tested with Scala 3.1.2 and the behaviour is the same (works fine in the ScalaPy tests, but not if ScalaPy is used as a dependency).

**Comment by shadaj** [comment]
Should be fixed with #278!

**Comment by sbrunk** [comment]
Awesome! I can confirm it's working with `0.5.1+14-d191d256`. Thanks @shadaj.
